resolver/src/lookup_state.rs

```diff
@@ -284,9 +284,7 @@ impl<C: DnsHandle<Error = ResolveError> + 'static> QueryFuture<C> {
         if was_cname {
             let next_query = Query::query(search_name, self.query.query_type());
             Ok(Async::Ready(self.next_query(
-                next_query,
-                cname_ttl,
-                response,
+                next_query, cname_ttl, response,
```

Comment:
IMO it would be better to remove these formatting-only changes from the PR and do the formatting in a separate PR before or after.

Reply:
Yeah my editor is aggressive on that. If you have concerns I can disable and do that.

rustfmt just runs as part of the RLS...